seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-tidb/pom.xml

```diff
@@ -60,6 +60,10 @@
                     <groupId>mysql</groupId>
                     <artifactId>mysql-connector-java</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.apache.spark</groupId>
+                    <artifactId>spark-sql_${scala.binary.version}}</artifactId>
+                </exclusion>
```

Comment:
Even if the version is excluded, it should be placed in the root-pom for unified management.

Reply:
In the root pom, the dependency of tispark-assembly version of 2.4.1, maybe it's indirect dependency of spark-sql version is right for current  root spark-sql version, but the version of tispark-assembly not bind to the root pom of seatunnel, it's a risk I think. 

exclude the dependency of spark-sql from tispark-assembly will reduce the risk, no matter how.

Reply:
> Even if the version is excluded, it should be placed in the root-pom for unified management
+1

Reply:
In my knowledge, whether you exclude the `spark-sql_ ` here, will not affect the version of `spark-sql_ ` in tidb.

Reply:
> In my knowledge, whether you exclude the `spark-sql_ ` here, will not affect the version of `spark-sql_ ` in tidb.

+1 , I forgot this when I review. My fault.

Reply:
> In my knowledge, whether you exclude the `spark-sql_ ` here, will not affect the version of `spark-sql_ ` in tidb.

When the indirect dependency of spark-sql's version in  tispark-assembly , is different from the root pom of spark-sql,
when read parquet data in some plugin, eg: hudi source, will throw the exception below:
----------------
 Caused by: org.apache.spark.sql.AnalysisException: Multiple sources found for parquet (org.apache.spark.sql.execution.datasources.v2.parquet.ParquetDataSourceV2, org.apache.spark.sql.execution.datasources.parquet.ParquetFileFormat), please specify the fully qualified class name.
-----------------

this issue is cause by the tispark-assembly's spark-sql has the ParquetDataSourceV2 and the root of spark-sql has the ParquetFileFormat, where read parquet datasource ,spark can not have a decision  using which Parquet format

This occurs in my case .

Reply:
Could you please use `mvn dependency:tree` to print your `sparl_sql` version?